mrp_production_component_report_xlsx/readme/USAGE.md

```
@@ -0,0 +1,5 @@
+    Go to the "Manufacturing > Operations > Manufacturing Orders"
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)